src/OpenTK.GLWidget/GLWidget.cs

```diff
@@ -26,99 +26,132 @@
 #endregion
 
 using System;
-using System.Collections.Generic;
-using System.Runtime.InteropServices;
-using System.Security;
 using System.Threading;
 using System.ComponentModel;
 
 using OpenTK.Graphics;
 using OpenTK.Platform;
-using OpenTK;
 
 using Gtk;
+using OpenTK.OSX;
+using OpenTK.Win;
+using OpenTK.X11;
 
 namespace OpenTK
 {
+    /// <summary>
+    /// The <see cref="GLWidget"/> is a GTK widget for which an OpenGL context can be used to draw arbitrary graphics.
+    /// </summary>
     [ToolboxItem(true)]
-    public class GLWidget: DrawingArea, IDisposable
+    public class GLWidget: DrawingArea
```

Comment:
There may be applications in which you'd like to extend the widget - overriding other methods of the underlying widget to enable it to work in other use cases, for instance.